code/provision/playbooks/roles/rke-kubernetes/templates/rke_cluster.yml

```diff
@@ -0,0 +1,52 @@
+---
+cluster_name: "{{ cluster_name }}"
+
+kubernetes_version: "{{ rke_kubernetes_version }}"
+
+bastion_host:
+  address: "{{ bastion_ip_address }}"
+  port: "{{ ssh_port }}"
+  user: "{{ rke_ssh_user }}"
+  #ssh_key: ""
+  ssh_key_path: "{{ ssh_key_path }}"
+
+# Currently only nginx ingress provider is supported.
+# To disable ingress controller, set `provider: none`
+# `node_selector` controls ingress placement and is optional
+ingress:
+    provider: none
+    node_selector:
+      app: ingress
+
+ignore_docker_version: false
+
+services:
+  etcd:
+    backup_config:
+      interval_hours: 24
+      retention: 6
+  kubelet:
+    extra_binds:
+      - "/sbin/apparmor_parser:/sbin/apparmor_parser"
```

Comment:
This is quite a neat solution, is this the only change needed to make apparmor work fine?

Reply:
Yes, that's the only change needed